pulsar-client/src/main/java/org/apache/pulsar/client/impl/Backoff.java

```diff
@@ -39,7 +39,7 @@ public Backoff(long initial, TimeUnit unitInitial, long max, TimeUnit unitMax) {
     public long next() {
         long current = this.next;
         if (current < max) {
-            this.next = Math.min(this.next * 2, this.max);
+            this.next = Math.min(this.next + this.next/2, this.max);
         }
 
         // Randomly increase the timeout up to 25% to avoid simultaneous retries
```

Comment:
@jai1 Would it be worth to maybe also play with this 25% random parameter? Should we rather subtract some ~10% instead of adding time?